package.json

```diff
@@ -22,6 +22,13 @@
     "posttest": "npm run lint",
     "mocha": "./node_modules/.bin/_mocha --timeout 40000 --require strong-mocha-interfaces --require test/init.js --ui strong-bdd"
   },
+  "files": [
+    "intl",
+    "lib",
+    "docs.json",
+    "index.js",
+    "setup.sh"
```

Comment:
cloudant contains more setup files I think:
setup.sh
test.setup.sh
test.js
docker.setup.js
docker-compose.yaml

Reply:
on a second thought, I am not sure do we need to include the setup files when install the connector as a dependency. usually the app should provide a valid db. So probably remove the `setup.sh` file?

Reply:
back from discussion in 
I see why `setup.sh` is needed in other connectors.
Tip: lb-next/repository-tests has standalone setup script for cloudant, see https://github.com/strongloop/loopback-next/blob/master/acceptance/repository-cloudant/setup.sh, so including/excluding it in `files` doesn't affect the tests. I will honor @emonddr 's decision on it.

Reply:
> back from discussion in https://github.com/strongloop/loopback-connector-mysql/pull/415/files#r391873007
> I see why `setup.sh` is needed in other connectors.
> Tip: lb-next/repository-tests has standalone setup script for cloudant, see https://github.com/strongloop/loopback-next/blob/master/acceptance/repository-cloudant/setup.sh, so including/excluding it in `files` doesn't affect the tests. I will honor @emonddr 's decision on it.

I figure we should be consistent in what we include in "files" section.